### ISSUE TITLE
Little enhancements for reduced CSS situations

### ISSUE THREAD

**Issue by querkmachine** [querkmachine/beeps.website#374]
Just things I noticed whilst dossing about in Lynx.

- [x] Add horizontal rule between navigation and start of content
- [x] Add horizontal rule between end of content and footer
- [x] No homepage link because no logo
- [x] Missing space on publish date on blog cards
- [x] Format button lists as lists
- [ ] It's actually a lil weird that published/updated dates appear before the headings...

**Comment by querkmachine** [comment]
Gonna sit on the dates-relative-to-heading one for now as it requires a bit more rework than all the other quick wins.
